actionview/lib/action_view/test_case.rb

```diff
@@ -42,6 +42,176 @@ def initialize
       end
     end
 
+    module Parsing
```

Comment:
Since we are not going to conditionally include this module anymore, let's move to a new file.